lib/components/Button.js

```diff
@@ -68,60 +68,65 @@ const Button = React.forwardRef((props, ref) => {
     iconPosition == "left" ? "marginRight" : "marginLeft";
 
   return (
-    <Tooltip {...tooltipProps} disabled={!tooltipProps}>
-      <Parent
-        ref={ref}
-        onClick={handleClick}
-        className={classnames("neeto-ui-btn", [className], {
-          "neeto-ui-btn--style-primary": style === BUTTON_STYLES.primary,
-          "neeto-ui-btn--style-secondary": style === BUTTON_STYLES.secondary,
-          "neeto-ui-btn--style-danger": style === BUTTON_STYLES.danger,
-          "neeto-ui-btn--style-text": style === BUTTON_STYLES.text,
-          "neeto-ui-btn--style-link": style === BUTTON_STYLES.link,
-          "neeto-ui-btn--size-large": size === BUTTON_SIZES.large,
-          "neeto-ui-btn--width-full": fullWidth,
-          "neeto-ui-btn--icon-left": iconPosition == ICON_POSITIONS.left,
-          "neeto-ui-btn--icon-only": !label,
-          disabled: disabled,
-        })}
-        disabled={disabled}
-        {...elementSpecificProps}
-        {...otherProps}
-      >
-        {label && <span>{label}</span>}
+    <Tooltip
+      {...tooltipProps}
+      disabled={!tooltipProps || tooltipProps.disabled}
+    >
```

Comment:
```suggestion
    <Tooltip
      disabled={!tooltipProps}
      {...tooltipProps}
    >
```
@abhirampai _a can you try this instead?

Reply:
@ajmaln yes this works.